components/output/pca9685.rst

```diff
@@ -81,6 +82,27 @@ Configuration variables:
   Use this if you have multiple PCA9685s you want to use at the same time.
 - All other options from :ref:`Output <config-output>`.
 
+
+Note
```

Comment:
I don't like the title here being `Note`...
But cannot think of what it should be

Reply:
Yes. Good question. I tried to stick to the existing structure. Here are a few suggestions:

Usage
Example
Use case

Reply:
```suggestion
Examples
```